packages/react-workers/src/utils/sorting.js

```diff
@@ -6,8 +6,12 @@ import { firstBy } from 'thenby';
  * @param {object} [sortOptions]
  * @param {string | string[] | object[]} [sortOptions.sortBy] - One or more columns to sort by
  * @param {string} [sortOptions.sortByDirection] - Direction by the columns will be sorted
+ * @param {string} [sortOptions.sortByColumnType] - Column type
```

Comment:
I think we should enum here the specific allowed types, not just 'string' generic type, to make evident to clients what values are ok.